### PR TITLE
Added character count support to input box

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-modern-theme](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-modern-theme](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-modern-theme)

File: src/app/public/modules/input-box/fixtures/input-box.component.fixture.html [hunk 1]
```diff
@@ -220,3 +220,28 @@
...
+    >
+    </sky-character-counter-indicator>
+    <input
+      class="sky-form-control"
+      skyCharacterCounter
+      skyId
+      [skyCharacterCounterIndicator]="characterCount"
+      [skyCharacterCounterLimit]="10"
+      #characterCountInput="skyId"
+    >f
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/input-box/fixtures/input-box.component.fixture.html:245]: typo here? `f`

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
I noticed when you're in modern theme, there's a slight delay in the transition of the label color, when compared to the transition of the focus/border color. Its most noticeable in dark mode. Is that intentional?

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-modern-theme](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-modern-theme)

File: src/app/public/modules/input-box/fixtures/input-box.component.fixture.html (hunk 1 above)
Comment (Blackbaud-PaulCrowder) [reply src/app/public/modules/input-box/fixtures/input-box.component.fixture.html]: Oops. Fixed.

**Comment by Blackbaud-PaulCrowder** [comment]
> I noticed when you're in modern theme, there's a slight delay in the transition of the label color, when compared to the transition of the focus/border color. Its most noticeable in dark mode. Is that intentional?

It's not intentional, but not something that's a huge deal right now. Once we get all the components themed, I can circle back and look at tidying up some of these minor issues.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-modern-theme](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-modern-theme)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Comment by Blackbaud-PaulCrowder** [comment]
#105

**Pull request merged**